#: component.py:90
msgid "NextGIS geoinformation system"
msgstr "Геоинформационная система NextGIS"

#: component.py:269
msgid "Linux kernel"
msgstr "Ядро Linux"

#: component.py:272
msgid "OS distribution"
msgstr "Дистрибутив ОС"

#: component.py:284
msgid "CPU"
msgstr "Процессор"

#: component.py:287
msgid "RAM"
msgstr "ОЗУ"

#: exception.py:87
msgid "Validation error"
msgstr "Ошибка валидации"

#: exception.py:92
msgid "Forbidden"
msgstr "Запрещено"

#: exception.py:97
msgid "Insufficient permissions"
msgstr "Недостаточно прав"

#: exception.py:102
msgid "Operational error"
msgstr "Операционная ошибка"

#: exception.py:107
msgid "External service error"
msgstr "Ошибка внешнего сервиса"

#: exception.py:111
msgid "Not configured"
msgstr "Не сконфигурировано"

#: storage.py:289
msgid "Storage full"
msgstr "Хранилище переполнено"

#: storage.py:290
msgid ""
"This operation can't be performed because the storage limit (%s) has been "
"already exceeded by %s."
msgstr ""
"Операция не может быть выполнена, так как лимит использования хранилища (%s) "
"уже превышен на %s."

#: storage.py:300
msgid "Not enough storage"
msgstr "Недостаточно места в хранилище"

#: storage.py:301
msgid ""
"This operation requires %s of storage and can't be performed because only %s "
"available."
msgstr ""
"Операция не может быть выполнена, так как для ее выполнения требуется %s в "
"хранилище, однако доступно только %s."

#: storage.py:311
msgid "Estimation is already running"
msgstr "Процесс оценки хранилища уже запущен"

#: test/test_exception.py:76
msgid "The answer is %d"
msgstr "Ответ %d"

